### PR TITLE
Update n8n release to 0.213.0

### PR DESCRIPTION
**Pull request by juicygoose**
## Summary
Update the n8n release to 0.213.0. Introducing new features and bugfixes
https://docs.n8n.io/reference/release-notes/

## Test Plan
NA

## Checklist

- [x]  No images hosted from dockerhub
- [x]  Are dashboards present to understand the health of the application.  There **must** be at least 1 of these
    - [x]  all databases should have dashboards
    - [x]  ideally also have at least cpu/mem utilization dashboards for webserver tier of the app
    - [x]  you can use `plural from-grafana` to convert a grafana dashboard found via google to our CRD
- [x]  Are scaling runbooks present
    - [x]  all databases **must** have scaling runbooks
    - [x]  you can use the charts in `pluralsh/module-library` to accelerate this
- [x]  do you need to add config overlays?
    - [x]  inputing secrets
    - [x]  configuring autoscaling
- [x]  If there’s a web-facing component to the app, we need to support OIDC authentication and setting up private networks if no authentication option is viable
- [x]  All major clouds must be supported
    - [x]  Azure
    - [x]  AWS
    - [x]  GCP

### CODE REVIEW

File: n8n/helm/n8n/Chart.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@ name: n8n
 description: helm chart for n8n
 type: application
 version: 0.1.6
```
Comment (DavidSpek) [inline n8n/helm/n8n/Chart.yaml:5]: Can you bump the chart version? This is required to push a new helm chart to Plural.

File: n8n/helm/n8n/Chart.yaml [hunk 2]
```diff
@@ -3,8 +3,8 @@ name: n8n
...
-appVersion: "0.202.1"
+appVersion: "0.212.1"
 dependencies:
-- name: postgres
```
Comment (DavidSpek) [inline n8n/helm/n8n/Chart.yaml:8]: It's not very important, but correct yaml formatting actually doesn't have an indent for array items.

File: n8n/helm/n8n/values.yaml [hunk 3]
```diff
@@ -8,7 +8,7 @@ image:
   repository: dkr.plural.sh/n8n/n8nio/n8n
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: "0.202.1"
+  tag: "0.212.1"
```
Comment (DavidSpek) [inline n8n/helm/n8n/values.yaml:11]: It looks like the latest version is `0.213.0` based on the image we have on our repository.

**Review by DavidSpek — COMMENTED** [review]
Overall it looks good, just a few small comments and nits that can be ignored.

File: n8n/helm/n8n/Chart.yaml (hunk 1 above)
Comment (juicygoose) [reply n8n/helm/n8n/Chart.yaml]: done

File: n8n/helm/n8n/Chart.yaml (hunk 2 above)
Comment (juicygoose) [reply n8n/helm/n8n/Chart.yaml:8]: done

File: n8n/helm/n8n/values.yaml (hunk 3 above)
Comment (juicygoose) [reply n8n/helm/n8n/values.yaml]: done !

**Review by DavidSpek — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**